### DISCUSSION TITLE
Admin order - to show only enabled products in the list "Please Select Products to Add"

### DISCUSSION THREAD

**Discussion by tomekjordan** [OpenMage/magento-lts#2108] [Ideas]
When I try to create order as admin "Please Select Products to Add"
Products selector shows all products in the store, even disabled.
Its a waste of time to find proper products to add to order. Default option should load just enabled products.
Probably the same "issue" appears in any other places where list of products are loaded in backend.
If admin disabled the product - it means it not used in the store (temporary or permamently)

**Comment by rafaelpatro** [comment]
I think displaying _disabled products_ is useful when you want a shipping quote. You can't do the same in the frontend.

**Comment by colinmollenhour** [comment]
Isn't this what the Visibility attribute is for, though?
